src/test/java/teammates/test/cases/logic/InstructorsLogicTest.java

```diff
@@ -728,4 +730,25 @@ private void verifySameInstructor(InstructorAttributes instructor1, InstructorAt
         assertEquals(instructor1.email, instructor2.email);
     }
 
+    private void testGetCoOwnersForCourse() {
+        ______TS("Verify co-owner status of generated co-owners list");
+        String courseId = "idOfTypicalCourse1";
+        List<InstructorAttributes> generatedCoOwners = instructorsLogic.getCoOwnersForCourse(courseId);
+        for (InstructorAttributes generatedCoOwner : generatedCoOwners) {
+            assertTrue(generatedCoOwner.hasCoownerPrivileges());
+        }
+
+        ______TS("Verify all co-owners present in generated co-owners list");
+
+        List<InstructorAttributes> instructors = instructorsLogic.getInstructorsForCourse(courseId);
```

Comment:
> I think it may be better to generate it from the dataBundle.

I'm not entirely sure how to do this 😅. Any suggestions on how I can get all instructors for a particular course? As of now I have gotten `dataBundle.instructors.values()` which gives me all the possible instructors in the DB. Will I have to loop through these and pick all the ones in the course I'm using to test this method? Or is there a better way?